recipe/recipe.yaml

```
@@ -18,14 +18,18 @@ build:
 
 requirements:
   host:
-    - python
+    - python ${{ python_min }}
     - pip
     - setuptools
   run:
-    - python
+    - python >=${{ python_min }}
     - numba >=0.59.1
 
 tests:
+  - requirements:
+     run:
+       - python ${{ python_min }}
+       - pip
   - python:
       imports:
       - numba_cuda
```

Looks like we would need to convert this to `script` as well

```suggestion
  - script:
      - python -m pip check
      - python -c "import numba_cuda"
```

Note: This includes the `pip_check` below, but was unable to grab that in the current diff